### PR TITLE
CXP-1364 Update the product grid page

### PR DESCRIPTION
**Pull request by RiffVentura**
image

- Updated the product display grid: removed filters and search bar
- The products displayed are either queried with  catalog mapped products endpoint or regular catalog product endpoint 
- Patched a bug where an unauthorized error would not clear the access token
- Updated fixtures JSON files list with more readable names 

Note on the UI: products are not entirely aligned with the left edge of the blue helper in favor more centered approach

### CODE REVIEW

File: assets/styles/base/_base.scss [hunk 1]
```diff
@@ -44,5 +44,5 @@ a {
 }
 
 .connected-base {
-  padding: 31px 100px 50px 50px;
+  padding: 31px 100px 50px;
 }
```
Comment (hogren) [inline assets/styles/base/_base.scss:48]: All code I saw in the demo app is in the form :

    padding-top: xxx
    padding-left: xxx
    […]

I think that is a good thing for the readability. What do you think about that ?

File: assets/styles/components/_breadcrumb.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    color: variables.$color-grey-120;
+  }
+
+  &__tail {
+    color: variables.$color-grey-100;
+  }
+
+  & > *:not(:last-child)::after {
+    display: inline-block;
+    margin: 0 5px;
```
Comment (hogren) [inline assets/styles/components/_breadcrumb.scss:18]: I propose again to use the multi-line form with `margin-top` and `margin-right`. It's not very important, but it's for harmonization and readability.

File: assets/styles/pages/_products.scss [hunk 3]
```diff
@@ -12,6 +12,12 @@
       color: variables.$color-demo-blue;
     }
 
+    &__header {
+      .page-title {
+        margin: 21px 0 33px;
```
Comment (hogren) [inline assets/styles/pages/_products.scss:17]: If you read my precedent comments, think about this line too.

**Review by hogren — APPROVED** [review]

File: src/PimApi/PimCatalogApiClient.php [hunk 4]
```diff
@@ -37,16 +38,39 @@ private function getClient(): HttpClientInterface
...
+        }
+
+        return $pimUrl;
+    }
+
+    /**
+     * @throws PimApiUnauthorizedException
+     * @throws PimApiException
+     */
+    private function throwOnErroneousResponse(int $expectedCode, int $actualCode, string $message): void
```
Comment (guillaumefer-akeneo) [inline src/PimApi/PimCatalogApiClient.php:55]:
```suggestion
    private function throwCustomException(int $expectedCode, int $actualCode, string $message): void
```
`throwOnErroneousResponse` is hard to read/understand IMHO.

**Review by guillaumefer-akeneo — APPROVED** [review]

File: src/PimApi/PimCatalogApiClient.php (hunk 4 above)
Comment (RiffVentura) [reply src/PimApi/PimCatalogApiClient.php:55]: Would `throwOnErrorCode` be ok ?

File: assets/styles/base/_base.scss (hunk 1 above)
Comment (RiffVentura) [reply assets/styles/base/_base.scss:48]: Mixed feelings:
On the one hand yes it's not easily obvious with the shorthand version what are the parameter order
On the other, it is something simple, and our CSS linter prefers the shorthand version.
So I don't know and want other's opinions on it as well

File: src/Controller/ShowCatalogAction.php [hunk 5]
```diff
@@ -4,63 +4,70 @@
...
-        if ($catalog->enabled) {
-            $products = $this->fetchProductsQuery->fetch($locale, $catalog->id);
-        } else {
-            $products = [];
-        }
+        $catalog = $this->getCatalogBy($catalogId);
+        $products = $this->getProductsFrom($catalog);
 
         return new Response(
             $this->twig->render('products.html.twig', [
```
Comment (masacc) [inline src/Controller/ShowCatalogAction.php:38]: to be consistent: 
```suggestion
            $this->twig->render('catalog.html.twig', [
```

File: tests/Integration/PimApi/PimCatalogApiClientTest.php [hunk 6]
```diff
@@ -206,4 +207,83 @@ public function itThrowsWhenItFailsToSetProductMappingSchema(): void
...
+
+        $result = $this->pimCatalogApiClient->getMappedProducts('70313d30-8316-41c2-b298-8f9e7186fe9a');
+
+        $this->assertEmpty($result);
+    }
+
+    /**
+     * @test
+     */
+    public function itThrowsWhenAnErrorOccurOnMappedProductsRetrieval(): void
```
Comment (masacc) [inline tests/Integration/PimApi/PimCatalogApiClientTest.php:263]:
```suggestion
    public function itThrowsWhenAnErrorOccursOnMappedProductsRetrieval(): void
```

File: tests/Integration/PimApi/PimCatalogApiClientTest.php [hunk 7]
```diff
@@ -206,4 +207,83 @@ public function itThrowsWhenItFailsToSetProductMappingSchema(): void
...
+        );
+
+        $this->expectException(PimApiException::class);
+        $this->pimCatalogApiClient->getMappedProducts('8a8494d2-05cc-4b8f-942e-f5ea7591e89c');
+    }
+
+    /**
+     * @test
+     */
+    public function itThrowsWhenCatalogDisablingErrorOccurOnMappedProductsRetrieval(): void
```
Comment (masacc) [inline tests/Integration/PimApi/PimCatalogApiClientTest.php:279]:
```suggestion
    public function itThrowsWhenCatalogDisablingErrorOccursOnMappedProductsRetrieval(): void
```

**Review by masacc — APPROVED** [review]

File: assets/styles/base/_base.scss (hunk 1 above)
Comment (RiffVentura) [reply assets/styles/base/_base.scss]: The team consensus is to keep shorthand versions

**Pull request merged**